var/spack/repos/builtin/packages/hdf/package.py

```diff
@@ -66,4 +65,5 @@ def install(self, spec, prefix):
         configure(*config_args)
 
         make()
-        make("install")
+        make('check')
```

Comment:
I haven't been so far. I'm still hoping for #1186 to be merged someday, making all of this obsolete.